cluster/examples/kubernetes/ceph/operator-openshift.yaml

```
@@ -402,6 +402,8 @@ data:
   ROOK_ENABLE_DISCOVERY_DAEMON: "false"
   # Enable volume replication controller
   CSI_ENABLE_VOLUME_REPLICATION: "false"
+  # The timeout value (in seconds) of Ceph commands. It should be >= 1. If this variable is not set or is an invalid value, it's default to 15.
+  ROOK_CEPH_COMMANDS_TIMEOUT: "15"
```

Adding the seconds suffix helps it be clear
```suggestion
  ROOK_CEPH_COMMANDS_TIMEOUT_SECONDS: "15"
```